### DISCUSSION TITLE
what-does-he-eat/

### DISCUSSION THREAD

**Discussion by giscus** [eallion/eallion.com#113] [Giscus]
# what-does-he-eat/

奈飞版《三体》第一季的结尾定格在把云天明送入太空这段情节中。这是我最喜欢的一段情节之一。《三体》我读了两遍，2016 年开车通勤时在喜马拉雅听了一遍。之后又读了一遍。全书让我感怀至深，久久无法忘怀的，就是维德故意告诉程心是云天明送她星星后，他们却“杀死”了他，决定“只送大脑”时，然后她呆呆地说的那句话：——“那，他吃什么？！”

https://www.eallion.com/what-does-he-eat/

**Comment by koobai** [comment]
这是自己看完网飞版的感受：没看过原著，听过解说，看过腾讯版。演员方便，腾讯版的叶文洁太喜欢了，网飞的啥玩意。程心也是，不喜欢这演员，想象中应该更白莲花点。但整体剧情网飞版本可以看，腾讯版本太慢了。

**Reply by eallion** [reply]
我现在不听解说了，不管是小说的还是电影的，这东西害人不浅。一部作品值得看，那就值得花时间看，不值得看的，花一点点时间也不值得看。

**Reply by koobai** [reply]
我是喜欢看已经看过的影视解说，能发现很多当时自己看的时候没发现的细节。太佩服这些解说的人了，够细致够有文化。
另外就是喜欢看一些老剧的解说

**Reply by eallion** [reply]
对，这种我也喜欢看。有时候还会专门去看他们的解说。

**Comment by Adek06** [comment]
网飞版确实拍得足够优秀，无论剧情人物改编还是拍摄手法，明显高了腾讯版几个层级

**Reply by eallion** [reply]
整体来说挺不错的。

**Comment by reds2lucas** [comment]
老牛，要给你博客留言还真不容易。github国内不容易连上啊哈哈。

**Reply by reds2lucas** [reply]
偷看你博客也有些年头了，今天特地用steam++ 加速了github 总算是登录上去了

**Reply by eallion** [reply]
现在评论也要谨慎，所以就加高一点门槛了。谢谢关注！

**Reply by reds2lucas** [reply]
你现在是在深还是在哪呀？

**Reply by eallion** [reply]
疫情时离深了。

**Reply by reds2lucas** [reply]
那你猜猜我是谁

**Reply by eallion** [reply]
很有可能是以前一起看越狱的一位故人。

**Reply by reds2lucas** [reply]
卧槽，猜的这么准

**Comment by wangyunzi** [comment]
你的博客怎么这么好看了

**Reply by eallion** [reply]
别人的模板写得好，模板写得好。。
